packages/devtools-launchpad/src/components/LandingPage.css

```diff
@@ -93,6 +93,10 @@
   transition: var(--base-transition);
 }
 
+.landing-page .panel header input[type=button] {
+  flex: none;
+}
```

Comment:
@nchevobbe updated! The padding is inherited and looks ok, just had to override the inherited flex size.